snap/hooks/configure

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh -e
```

Comment:
Since these are copied from upstream without change, did you want to retain your script for syncing them?  That might help us keep up to date in future?

Reply:
I thought about that and I think it will be very rare that someone outside of Canonical will change the scripts to build the charm. IMO I think this repo should be the source of "truth" to anything related to the snap package instead of doing on the upstream repo.

Regarding the name, it's unusual to have the `snap` at the name so I'm in favor of keeping this way.

cc: @aieri

Reply:
On the stand up meeting Andrea agrees with your proposal, so I'm going to change the name and sync the script from the upstream